packages/material-ui/src/AccordionSummary/AccordionSummary.js

```diff
@@ -1,74 +1,127 @@
 import * as React from 'react';
 import PropTypes from 'prop-types';
 import clsx from 'clsx';
+import { deepmerge } from '@material-ui/utils';
+import { unstable_composeClasses as composeClasses } from '@material-ui/unstyled';
+import experimentalStyled from '../styles/experimentalStyled';
+import useThemeProps from '../styles/useThemeProps';
 import ButtonBase from '../ButtonBase';
-import withStyles from '../styles/withStyles';
 import AccordionContext from '../Accordion/AccordionContext';
+import accordionSummaryClasses, {
+  getAccordionSummaryUtilityClass,
+} from './accordionSummaryClasses';
 
-export const styles = (theme) => {
+const overridesResolver = (props, styles) => {
+  const { styleProps } = props;
+
+  return deepmerge(styles.root || {}, {
+    ...(styleProps.expanded && styles.expanded),
+    [`&.${accordionSummaryClasses.focusVisible}`]: styles.focusVisible,
+    ...(styleProps.disabled && styles.disabled),
+    [`& .${accordionSummaryClasses.content}`]: styles.content,
+    [`& .${accordionSummaryClasses.expandIconWrapper}`]: styles.expandIconWrapper,
+  });
+};
+
+const useUtilityClasses = (styleProps) => {
+  const { classes, expanded, disabled } = styleProps;
+
+  const slots = {
+    root: ['root', expanded && 'expanded', disabled && 'disabled'],
```

Comment:
```suggestion
    root: ['root', expanded && 'expanded', disabled && 'disabled'],
    focusVisible: ['focusVisible'],
```

Reply:
We expect that it exists on line https://github.com/mui-org/material-ui/pull/24540/files#diff-a3e45615b0516b905db6b26935126e30763f45b8901b2bee960adb5c3be2b997R152